test/Common/HttpApiHandlerTests.cs

```diff
@@ -11,6 +11,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using DurableTask.Core;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.ContextInterfaces;

```

Comment:
We can probably remove all of these additional `using` statements once you change the interface namespaces.